sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -62,15 +62,17 @@ case class CreateViewCommand(
     comment: Option[String],
     properties: Map[String, String],
     originalText: Option[String],
-    child: LogicalPlan,
+    analyzedPlan: LogicalPlan,
```

Comment:
how do we analyze it since it's not a child?

Reply:
The analyzed plan is passed here. I can keep the name `child` if that's clearer.